pyk/src/tests/integration/cterm/test_simple.py

```diff
@@ -60,6 +122,10 @@ def config(kprint: KPrint, k: str, state: str, constraint: str | None = None) ->
             (_constraint,),
         )
 
+    @staticmethod
+    def constraint(kprint: KPrint, c: str) -> KInner:
+        return kprint.parse_token(KToken(c, BOOL))
```

Comment:
Where is this called?

Reply:
It's not, I'll remove it, I think I was using as a helper to print some debugging information for the test.